SportAlert.php

```diff
@@ -1,103 +1,121 @@
+	// teamNameInTable($myTeam);
+	// $array = getAllGamesForTeam($myTeam);
+	// $array = gamesToday($array);
+	// print("Today there is a ".$array[0]["TeamName"]." game in ".$array[0]["Location"]."!<br/>");
+	// print("Be sure to tune in for the game at ".$array[0]["Time"].".<br/>");
+
+	# TODO: GET USER PREFERENCES
+
+	# For each user
+	# 	Check user preference
+	# 	Check team data
+	# 	Respond if appropriate
+
+	$queryUsers = "SELECT * FROM `Users`";
+
+	# For each User:
+	if($resultUsers = mysqli_query($link, $queryUsers)) {
+		while($userData = mysqli_fetch_array($resultUsers)) {
+			
+			# Update email info
+			$to = $userData["Email"];
+			$message = "Hello ".$userData["UserName"].",\n";
+
+			# For each setting ID:
+			$queryUserSettings = "SELECT * FROM `UserSettings` WHERE `SettingID`='".$userData["SettingID"]."'";
+			if($resultUserSettings = mysqli_query($link, $queryUserSettings)) {
+				while($userSettingData = mysqli_fetch_array($resultUserSettings)) {
+					
+					# Get game information
+					$tName = $userSettingData["TeamName"];
+					if(teamNameInTable($tName)) {
+						$array = getAllGamesForTeam($tName);
+						if($array[0]["TeamName"] != "") {
+						    $message = $message."Today there is a ".$array[0]["TeamName"]." game in ".$array[0]["Location"]."!\n";
+						 	$message = $message."Be sure to tune in for the game at ".$array[0]["Time"].".\n";
+						}
+					}
 
-		if(mysqli_connect_error()) {
-			die("ERROR: Could not connect to database<br/>");
-		}
-		print("Connection Successful - DB_SportAlert<br/><br/>");
-	}
-	
-	/*
-		Given a team, search DB to see if valid
-	*/
-	function teamNameInTable($tName) {
-		global $link;
-		$query = "SELECT * FROM `Teams`";
-
-		if($result = mysqli_query($link, $query)) {
-			while($row = mysqli_fetch_array($result)) {
-				// print($row["TeamName"]);
-				if($row["TeamName"] == $tName) {
-					// print("Found My Team: ".$tName."<br/>");
-					return true;
 				}
 			}
-		}
-		else {
-			die("ERROR: teamNameInTable() query did not work<br/>");
-		}
-		return false;
-	}
-
-	/*
-		Given a team, get all games played by said team
-	*/
-	// TODO: explore alternative method of using 'join'
-	// $query ="SELECT * FROM `Games` inner join Teams on Games.TeamName = Teams.TeamName";
-	function getAllGamesForTeam($tName) {
-		global $link;
-		$query = "SELECT * FROM `Games` WHERE `TeamName`='".$tName."'";
-
-		if($result = mysqli_query($link, $query)) {
-			while($row = mysqli_fetch_array($result)) {
-				$array[] = $row;
+			else {
+				die("ERROR: unable to access user settings db<br/>");
 			}
-			// print_r($array);
-		}
-		else {
-			die("ERROR: getAllGamesForTeam() query did not work<br/>");
-		}
-		return $array;
-	}
 
-	/*
-		Given an array of games, find games for today
-	*/
-	function gamesToday($array) {
-		$date = date('Y-m-d');
-		for($count = 0; $count < sizeof($array); $count++) {
-			if($array[$count]["Date"] == $date) {
-				$array[] = $array[$count];
-			}
+			// # Send Notification
```

Comment:
Commenting this section out for now to quite actual email notifications. Use print() statements for development purposes at this time...